src/requests/application.ts

```diff
@@ -0,0 +1,97 @@
+import { gql, GraphQLClient } from "graphql-request";
+import type { ApplicationResponse } from "../types";
+
+export class ApplicationClient {
+  protected client: GraphQLClient;
+
+  constructor(client: GraphQLClient) {
+    this.client = client;
+  }
+
+  async bopsResponse(sessionId: string): Promise<ApplicationResponse | null> {
+    return bopsApplicationResponse(this.client, sessionId);
+  }
+
+  async emailResponse(sessionId: string): Promise<ApplicationResponse | null> {
+    return emailApplicationResponse(this.client, sessionId);
+  }
+
+  async uniformResponse(
+    sessionId: string
+  ): Promise<ApplicationResponse | null> {
+    return uniformApplicationResponse(this.client, sessionId);
+  }
+}
+
+export async function uniformApplicationResponse(
```

Comment:
I'm exporting `ApplicationClient` and also still exporting the raw functions for two reasons:
1. this pattern is consistent with what we already have
2. when we introduce a test suite for these request functions, this pattern should make mocking easier (depending on the approach)